test/SimpleVersion.Core.Tests/Pipeline/Formatting/Semver2FormatProcessFixture.cs

```diff
@@ -45,7 +45,7 @@ public static IEnumerable<object[]> LabelParts()
             if (parts.Length > 0)
                 expected = $"{version}-{string.Join(".", parts)}.{height}.4ca82d2";
             else
-                expected = $"{version}-4ca82d2+{height}";
+                expected = $"{version}-example.4ca82d2+{height}";

```

Comment:
Need to confirm the max length of a label for both semver1 and semver2 formats.  Ideally, providing a substring to ensure the max length is not breached.